ui/src/overridden/frontsite/Document/DocumentDetails/DocumentMetadataTabs/Identifiers.js

```diff
@@ -0,0 +1,211 @@
+import {
+  InfoPopup,
+  SeparatedList,
+  vocabularyApi,
+  withCancel,
+} from '@inveniosoftware/react-invenio-app-ils';
+import capitalize from 'lodash/capitalize';
+import _isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Divider, Table, Loader } from 'semantic-ui-react';
+
+export class Identifiers extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      vocabularyIdentifiers: null,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchVocabularies();
+  }
+
+  componentWillUnmount() {
+    this.cancellableFetch && this.cancellableFetch.cancel();
+  }
+
+  createQueryPromise = type => {
+    const query = vocabularyApi
+      .query()
+      .withType(type)
+      .qs();
+
+    return vocabularyApi.list(query);
+  };
+
+  serializer = hit => ({
+    value: hit.metadata.key,
+    text: hit.metadata.text,
+    url_path: hit.metadata?.data?.url_path,
+  });
+
+  query = async () => {
+    const {
+      data: { hits: identifierQuery },
+    } = await this.createQueryPromise('identifier_scheme');
+    const {
+      data: { hits: alternativeIdentifierQuery },
+    } = await this.createQueryPromise('alternative_identifier_scheme');
+
+    const combinedHits = identifierQuery.concat(alternativeIdentifierQuery);
+
+    return combinedHits.map(this.serializer);
+  };
+
+  fetchVocabularies = async () => {
+    this.cancellableFetch = withCancel(this.query());
```

Comment:
I have the feeling (maybe I am mistaken) that we are fetching each time... as discussed, it would be more efficient to fetch once and cache.

Reply:
Maybe that is a wrong interpretation on my part, how i understood it was that we did not want to make a separate call for each scheme and value, which does not happen. It does do 2 quick fetches when opening the tab itself, which in my opinion is really fast and light, so should be no problem. What do you think?

Reply:
why 2 fetches, not one ?

Reply:
because i have to fetch the identifiers + alternative identifiers

Reply:
I see, I forgot we had alternative identifiers there to. 
You could store it in redux so you could avoid the second request, but in this case I don't think it is crucial to do

Reply:
I think it is not ok: if I go and visit multiple books, e.g. 10 books, for each of them there will be an HTTP call.
If you cache it in Redux instead, you have the HTTP calls only for the first book I visit.

Reply:
done!